packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -80,13 +95,24 @@ class ButtonGroupButton extends React.Component {
     // The onFocus event doesn't get triggered in some browsers, hence, the focus state needs to be managed here.
     if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
       this.setState({ focused: true });
+      this.shouldShowFocus = true;
     }
 
     if (this.props.onKeyUp) {
       this.props.onKeyUp(event);
     }
   }
 
+  handleFocus(event) {
+    if (this.shouldShowFocus) {
+      this.setState({ focused: true });
+    }
+
+    if (this.props.onFocus) {
+      this.props.onFocus(event);
```

Comment:
the prop onFocus should be passed from ButtonGroup to here to avoid multiple onfocus events from getting called when Focus handler has added to both parent (ButtonGroup) and (ButtonGroupButton) as well.

Reply:
Updated here https://github.com/cerner/terra-core/pull/2933/commits/530e5fb55778e29b3c26ef4896915987ed6a7351